W09H01/src/test/pgdp/pools/TupleTest.java

```diff
@@ -22,15 +22,17 @@ void testEquals() {
     Integer int0 = 123;
     String string0 = "Ladner";
     
-    Tuple<Integer, String> tuple0 = new Tuple<>(int0, string0);
+    Tuple<Object, Object> tuple0 = new Tuple<>(int0, string0);
     Tuple<Integer, String> tuple0Duplicate = new Tuple<>(int0, string0);
+    Tuple<Object, Object> tuple0Reversed = new Tuple<>(string0, int0);
```

Comment:
I'd revert the change of generics in `testEquals()`. Two tuples with different generics types should still be comparable.
(Apologies, if this is getting kind of convoluted. I'm still very unfamiliar with git(hub).)